### ISSUE TITLE
Not tracking Anti-venom+ sips; Not tracking all house tele tab uses

### ISSUE THREAD

**Issue by BrennanHolm** [Dyldozer/SuppliesTrackerExternal#33]
Doesn't seem to be tracking anti-venom+ uses, even after consuming multiple and restarting the RuneLite client. Also, there seems to be an issue with it not logging all house tele tab uses either. It's counting some, but skips others.

**Comment by Dyldozer** [comment]
Haven't seen any issues, are you still experiencing this?

**Comment by BrennanHolm** [comment]
It tracks the first home tele, but nothing else in regards to that, and it's not tracking any of the anti-venom+ sips.

**Comment by wigglepop** [comment]
> Haven't seen any issues, are you still experiencing this?

i experience this when i click the heart icon below the xp drop icon. it does not track when you use that method for removing poison. clicking the venom pot does track it though.

![image](https://user-images.githubusercontent.com/77034846/104082176-f7128b80-526e-11eb-9453-b1b11b522bdd.png)

**Comment by wigglepop** [comment]
> > Haven't seen any issues, are you still experiencing this?
> 
> i experience this when i click the heart icon below the xp drop icon. it does not track when you use that method for removing poison. clicking the venom pot does track it though.
> 
> ![image](https://user-images.githubusercontent.com/77034846/104082176-f7128b80-526e-11eb-9453-b1b11b522bdd.png)


https://user-images.githubusercontent.com/77034846/104712791-f6419400-575d-11eb-868f-71b5f3922c57.mp4

**Comment by BrennanHolm** [comment]
Yes, I know now it doesn't track if you click the heart - you have to click the actual bottle. It's still not tracking home tele's outside the very first one. Any updates on the problem here?
